version.sbt

```diff
@@ -1 +0,0 @@
-ThisBuild / version := "2.0.2-SNAPSHOT"
```

Comment:
`version.sbt` was removed because sbt-ci-release handles the versioning https://github.com/sbt/sbt-ci-release#sbt .